packaging/windows/installer.nsi

```diff
@@ -205,9 +194,51 @@ Function .onInit
       ; At this point, I'm very much puzzled as why writing NSIS installer
       ; feels like reverse engineering a taiwanese NES clone...
       ExecWait '"$R0" /S _?=$R1'
+FunctionEnd
 
-    done:
+Function checkUninstaller
+  ; Check either 32-bit or 64-bit registry for parsec uninstaller
+  ReadRegStr $R0 ${PROGRAM_UNINST_ROOT_KEY} \
+    "${PROGRAM_UNINST_KEY}" \
+    "UninstallString"
+    StrCmp $R0 "" done
```

Comment:
```suggestion
  StrCmp $R0 "" done
```

I'm not sure, but it fells like the `StrCmp` is at the wrong indentation level :thinking:

Reply:
fixed!